### ISSUE TITLE
Error in lexedata.edit.add_segments

### ISSUE THREAD

**Issue by peterdekker** [Anaphory/lexedata#312]
https://github.com/openjournals/joss-reviews/issues/4140

When following the Tour of lexedata in the docs, I get an error when running lexedata.edit.add_segments:

```
python -m lexedata.edit.add_segments -q # doctest: +NORMALIZE_WHITESPACE
Cloning into '/tmp/tmp659j9cw4clts'...
fatal: unable to access 'https://github.com/cldf-clts/clts.git/': Could not resolve host: github.com
Traceback (most recent call last):
  File "/home/peter/git/lexedata-1.0.0/lenv/lib/python3.9/site-packages/cldfcatalog/config.py", line 67, in get_clone
    return pathlib.Path(self.clones[key])
  File "/usr/lib/python3.9/configparser.py", line 1254, in __getitem__
    raise KeyError(key)
KeyError: 'clts'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/peter/git/lexedata-1.0.0/src/lexedata/edit/add_segments.py", line 30, in <module>
    with cldfcatalog.Catalog.from_config("clts", tag="v2.0.0"):
  File "/home/peter/git/lexedata-1.0.0/lenv/lib/python3.9/site-packages/cldfcatalog/catalog.py", line 56, in from_config
    return cls(cfg.get_clone(key or cls.cli_name()), tag=tag)
  File "/home/peter/git/lexedata-1.0.0/lenv/lib/python3.9/site-packages/cldfcatalog/config.py", line 69, in get_clone
    raise KeyError('Config {0} has no entry for {1}'.format(self._files, key))
KeyError: 'Config /home/peter/.config/cldf/catalog.ini has no entry for clts'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/peter/git/lexedata-1.0.0/src/lexedata/edit/add_segments.py", line 43, in <module>
    clts_path.__enter__()
AttributeError: 'str' object has no attribute '__enter__'
```

I am running a development install (following CONTRIBUTING) of lexedata 1.0.0, and I did run `cldfbench catconfig` before, saying the default No to all downloads.

**Comment by peterdekker** [comment]
Similar error for add_concepticon:
```
python -m lexedata.edit.add_concepticon -q -l Name=en --add-concept-set-names --add-definitions
Traceback (most recent call last):
  File "/home/peter/git/lexedata-1.0.0/lenv/lib/python3.9/site-packages/cldfcatalog/config.py", line 67, in get_clone
    return pathlib.Path(self.clones[key])
  File "/usr/lib/python3.9/configparser.py", line 1254, in __getitem__
    raise KeyError(key)
KeyError: 'concepticon'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/peter/git/lexedata-1.0.0/src/lexedata/edit/add_concepticon.py", line 32, in <module>
    concepticon_path = cldfcatalog.Config.from_file().get_clone("concepticon")
  File "/home/peter/git/lexedata-1.0.0/lenv/lib/python3.9/site-packages/cldfcatalog/config.py", line 69, in get_clone
    raise KeyError('Config {0} has no entry for {1}'.format(self._files, key))
KeyError: 'Config /home/peter/.config/cldf/catalog.ini has no entry for concepticon'
```

**Comment by peterdekker** [comment]
And here as well:
```
python -m lexedata.report.segment_inventories -q --language Nzebi # doctest: +NORMALIZE_WHITESPACE
```

**Comment by Anaphory** [comment]
To be fixed in the upcoming PR.

 - I'm still confused why you encountered “access 'https://github.com/cldf-clts/clts.git/': Could not resolve host: github.com”, but I don't think I have a way to debug this, and with improved installation instructions and errors, I hope it will not be a problem any more.
 - I fixed the installation instructions, because “No” (`[y/N]`) is *not* a sensible default, contrary to what we had stated. With agreeing to cone the repositories, you would have not run into most of these issues.
 - Added a warning to `add_segments` about installing catalogs. It's the only script that needs a catalog at import-time, so it has a fall-back for when we build documentation on ReadTheDocs – and a normal user should be warned when they run into that. (`report.segment_inventories` imports it, so is thus also handled.)
 - Added a similar warning to `add_concepticon`
 -
